org.jacoco.doc/docroot/doc/maven.html

```diff
@@ -41,11 +41,26 @@
   the tests with the <tt>javaagent</tt> set and no coverage would be recorded.
 </p>
 
+<p class="hint">
+  When using the <tt>maven-surefire-plugin</tt> with custom <a href="http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#argLine">
+   <tt>argLine</tt></a> you need to append <tt>@{argLine}</tt> to the custom <tt>argLine</tt><br>
+   <tt>&lt;argLine&gt;-Xms1024m -Xmx2048m <b>@{argLine}</b>&lt;/argLine&gt;</tt>
+</p>
+
+<p class="hint">
+  For a multi-module maven project using <tt>maven-surefire-plugin</tt> with custom <a href="http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#argLine">
+  <tt>argLine</tt></a> and not running JaCoCo runtime agent in all modules, you can set an empty <tt>argLine</tt> property.
+  This will prevent the build from failing with message <tt>Error: Could not find or load main class @{argLine}</tt>.<br>
+  This is required because undeclared property is not expanded to empty string during <a href="http://maven.apache.org/surefire/maven-surefire-plugin/faq.html#late-property-evaluation">
+  late property evaluation</a> in <tt>maven-surefire-plugin</tt>.<br>
+  <tt>&lt;properties&gt;&lt;argLine&gt;&lt;/argLine&gt;&lt;/properties&gt;</tt>
+</p>
```

Comment:
We already have note about `argLine` on [page dedicated to `prepare-agent` mojo](http://www.eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html) page that is rendered from its [javadoc](https://github.com/jacoco/jacoco/blob/0988e8c083f34f84312056dc7782488c6e147504/jacoco-maven-plugin/src/org/jacoco/maven/AgentMojo.java#L40), which is btw also visible on `mvn jacoco:help`. So seems more logical to relocate this change to the javadoc. And only part about empty property is missing in it.